<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messenger</class>
 <widget class="QDockWidget" name="messenger">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetClosable</set>
  </property>
  <property name="windowTitle">
   <string>                                                                                                                                                        umbra | chat</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <widget class="Line" name="NameBar1">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>0</y>
      <width>54</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #c63333;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>260</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messageHistory">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>51</y>
      <width>581</width>
      <height>291</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>there's nothing here... say hi!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="messageBox">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>350</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>send a message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>426</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #c63333;</string>
    </property>
    <property name="text">
     <string>-&gt;</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="displayName1">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>0</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumGothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: right; padding: 5px; color: #eee;</string>
    </property>
    <property name="text">
     <string>Alice</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
